src/sentry/static/sentry/app/components/sidebar/sidebarMenuItem.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import {css} from '@emotion/core';
+
+import {Theme} from 'app/utils/theme';
+
+import SidebarMenuItemLink, {SidebarMenuItemLinkProps} from './sidebarMenuItemLink';
+import {OrgSummary} from './sidebarOrgSummary';
+
+type Props = {
+  children: React.ReactNode;
+} & SidebarMenuItemLinkProps;
+
+const SidebarMenuItem = ({to, href, children, ...props}: Props) => {
+  const hasMenu = !to && !href;
+  return (
+    <StyledSidebarMenuItemLink to={to} href={href} {...props}>
+      <MenuItemLabel hasMenu={hasMenu}>{children}</MenuItemLabel>
+    </StyledSidebarMenuItemLink>
+  );
+};
+
+export default SidebarMenuItem;
+
+export const menuItemStyles = (p: SidebarMenuItemLinkProps & {theme: Theme}) => css`
```

Comment:
Maybe we could not export it directly, but do a named export at the bottom of the file next to default one?